apps/backend/tests/test_services/test_fix_suggestion_engine.py

```
@@ -0,0 +1,376 @@
+
+    def test_validate_fix_safety_low_data_loss(self, engine, sample_df):
+        """Test fix validation with low data loss."""
+        fix = SuggestedFix(
+            transformation_type='drop_missing',
+            parameters={'columns': ['age']},
+            explanation='Drop rows with missing age',
+            estimated_data_loss=3.0,  # Below threshold
+            estimated_rows_affected=3,
+        )
+
+        is_safe, warnings = engine.validate_fix_safety(fix, sample_df)
```

Ruff (0.14.8)

Unpacked variable `is_safe` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)